src/labels/CustomLabels.labels

```diff
@@ -2207,6 +2207,14 @@
         <shortDescription>The RD2 Batch job can only be executed when Enhanced RD is enabled</shortDescription>
         <value>You must enable Enhanced Recurring Donations before you can run the Enhanced Recurring Donations batch job.</value>
     </labels>
+    <labels>
+        <fullName>RD2_ErrorRDIdIsMissingForCommitment</fullName>
```

Comment:
@sam-knox , this new custom label is used to verify that RD2 entry form does create and passes a Recurring Donation Id before a commitment request can be sent to Elevate API.

Reply:
@RadSFDeveloper - What circumstances would produce this error? Would it mean that the integration is broken?

Reply:
Or perhaps it would mean that the RD record itself couldn't be created (due to validation error or somthing)?

Reply:
If the RD is not created or there is an error either in the LWC or when passing the RD Id, the commitment creation must be prevented in such case. And even though this is a programatic error, this is another check to ensure data integrity and correct process.

Reply:
@sam-knox , that ^ is correct. Either validation or something happened that prevented RD creation or it was created by LWC is not passing it to the commitment creation. So if this happens (it should not) , the commitment request should not be sent to Elevate.

Reply:
@sam-knox , both suggestions are valid, chose the second one since it has more info about the cause and impact.